rpc/cc/arduino/cli/commands/v1/lib.proto

```diff
@@ -132,6 +132,8 @@ message LibraryDependencyStatus {
   string version_required = 2;
   // Version of the library dependency currently installed.
   string version_installed = 3;
+  // Version constraint of the dependency.
+  string version_constraint = 4;
 }
```

Comment:
`version_constraint` seems redundant here, why do we need a `version_constraint` if we already have a `version_required`?

I mean if we have, say:
```
version_required: 1.2.3
version_contraint: >=1.2.0 && <2.0.0
```
version `1.2.3` is already the solution and matches the `version_constraint`.